File: c#/c_recursive_calc/c_recursive_calc/Token.cs

namespace c_recursive_calc
{
    class Token
    {
        public enum TokenId { Number, Operator, Unknown, END }

        public string Text { get; set; }
        public TokenId Id { get; set; }
        
        public Token()
        {
            Text = "";
            Id = TokenId.END;
        }
    }
}

File: php/calc_parser/Token.php
<?php
/**
 * Created by PhpStorm.
 * User: SRigh
 * Date: 11.09.2015
 * Time: 13:18
 */
class Token
{
    public $tokenId = ["number"=>'number', 'operator'=>'operator','unknown'=>'unknown','end'=>'end'];
    public $text = " ";
    public $id = ['end'=>'end'];

    public function __constructor()
    {
        $this->text = "";
        $this->id = $this->tokenId['end'];
    }
}